docs: Add multiple values in whenChoiceValue and whenLabelValue

makseq opened the pull request:
### PR fulfills these requirements
- [x] Docs have been added/updated (for bug fixes/features)
<!-- ELLIPSIS_HIDDEN -->


----

| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit 99db19acd7c8119c4e9a062450be9151261e755d  | 
|--------|--------|

docs: support multiple values in whenChoiceValue and whenLabelValue

### Summary:
Update documentation and code to support multiple values for `whenChoiceValue` and `whenLabelValue`.

**Key points**:
- Update documentation to support multiple comma-separated values for `whenChoiceValue` and `whenLabelValue`.
- Modify `View.jsx` and `Choices.jsx` to handle multiple values.
- Update `docs/source/tags/view.md` and `docs/source/tags/choices.md` with new examples.
- Add examples and update parameter descriptions in `View.jsx` and `Choices.jsx`.


----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)



<!-- ELLIPSIS_HIDDEN -->

makseq committed:
docs: Add multiple values in whenChoiceValue and whenLabelValue

makseq committed:
Update View.jsx

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10882476552

hlomzik reviewed (changes requested):
This should also be added to Choices and Taxonomy

caitlinwheeless committed:
Edits and added to Choices tag

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10884749634

hlomzik commented on web/libs/editor/src/tags/control/Choices.jsx:
```diff
@@ -87,10 +87,10 @@ const { Option } = Select;
- * @param {string} [whenChoiceValue]   - Use with visibleWhen ("choice-selected" or "choice-unselected") and whenTagName, both are required. Narrow down visibility by choice value
+ * @param {region-selected|no-region-selected|choice-selected|choice-unselected} [visibleWhen] - Control visibility of the choices. Can also be used with the `when*` parameters below to narrow down visibility
+ * @param {string} [whenTagName]       - Use with `visibleWhen`. Narrow down visibility by name of the tag. For regions, use the name of the object tag, for choices, use the name of the `choices` tag
+ * @param {string} [whenLabelValue]    - Use with `visibleWhen="region-selected"`. Narrow down visibility by label value. Multiple values can be separated with commas
```
in all tags these props are coming from VisibilityMixin, so everything is good

hlomzik commented:
Taxonomy didn't have these parameters in docs, so we decided to leave it as is.

makseq committed:
Update View.jsx

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10913099405
